code/obj/machinery/espresso.dm

```diff
@@ -225,6 +228,10 @@
 								playsound(src.loc, 'sound/misc/pourdrink.ogg', 50, 1)
 						if ("Remove carafe")
 							if (!src.my_carafe)
+								user.show_text("The carafe is gone!")
+								return
+							if (get_dist(src, user) > 1 || isAI(user))
```

Comment:
Seems like this is using the old method of remote access check?

Reply:
I care that we are nearby and physical, so essentially the inversion of old method.